.github/DISCUSSION_TEMPLATE/issues-and-projects.yml

```diff
@@ -26,10 +26,10 @@ body:
   - type: checkboxes
     attributes:
       label: Select Feature Area
```

Comment:
You guys are the only help I've got I've been fighting for 2 years for my grandkids and I have known one but you guys   am getting drained stressed and worried them babe are my life.  If I can't get the evidence on them and I lose them  honestly I don't think I could live with out my grandkids and I read stuff last night that in my file that I can't have these CSS cps workers need to be stop save the children please